deployment/blackduck-alert.env

```diff
@@ -21,15 +26,14 @@ ALERT_ENCRYPTION_GLOBAL_SALT=
 
 # -- BlackDuck Provider Settings
 
+#ALERT_PROVIDER_BLACKDUCK_PROVIDER_COMMON_CONFIG_NAME=Default Config
```

Comment:
I know this is probably not part of this PR but why is this `COMMON_CONFIG_NAME` and not `DEFAULT_CONFIG_NAME`?

Reply:
COMMON_CONFIG_NAME for the environment variable gets turned into common.config.name which is the registered field name in the database for the configuration.  Each provider configuration you add has a common.config.name field.  Using this name requires no additional code to be added to AlertStartupInitializer to handle creating an initial Black Duck Provider configuration.  If we use a different environment variable name then we will need to add special logic to the AlertStartupInitializer class to process the blackduck provider and look for a specific environment variable name rather than use our existing environment variable naming convention.